### PR TITLE
streamline changest viewing for larger diffs

### PR DESCRIPTION
**Pull request by mattcosta7**
We basically output the same thing twice, and on large diffs we'll go well over the comment size limit for an issue. 

streamlining this by minimize the necessary output, removing the 'detailed' view, in favor of updating the changeset to contain added info about the file sizes

while here I also updated some dependencies

fixes https://github.com/github/webpack-bundlesize-compare-action/pull/296
fixes https://github.com/github/webpack-bundlesize-compare-action/pull/294
fixes https://github.com/github/webpack-bundlesize-compare-action/pull/293

### CODE REVIEW

**Comment by mattcosta7** [comment]
Once this lands, we should version and update internal consumers

File: __tests__/__snapshots__/main.test.ts.snap [hunk 1]
```diff
@@ -123,36 +123,18 @@ exports[`displays module information when files are added/removed/changed 1`] =
...
-\`src/client/this-is-another-file-with-a-long-name-to-test-the-table-sizing.tsx\` | 0 B | 1.04 KB | 🆕 +1.04 KB
-\`src/client/helpers/this-file-is-much-larger.ts\` | 1.35 KB | 13.01 KB | 📈 +11.66 KB (+865.77%)
-\`src/client/this-file-grew-larger.tsx\` | 562 B | 762 B | 📈 +200 B (+35.59%)
-\`src/client/helpers/this-file-grew-smaller.ts\` | 2.46 KB | 2.26 KB | 📉 -200 B (-7.94%)
-\`src/client/this-file-is-much-smaller.tsx\` | 11.67 KB | 10 B | 📉 -11.66 KB (-99.92%)
-\`src/client/routes.tsx\` | 2.72 KB | 0 B | 🔥 -2.72 KB (-100%)
-\`src/client/this-file-will-be-deleted.ts\` | 1.46 KB | 0 B | 🔥 -1.46 KB (-100%)
-\`src/client/render-memex.tsx\` | 1.04 KB | 0 B | 🔥 -1.04 KB (-100%)
-
-</details>
```
Comment (mattcosta7) [inline __tests__/__snapshots__/main.test.ts.snap:155]: this ouput was mostly duplicative, so just inlined some detail above to save on characters

File: src/parse-stats-file-to-json.ts [hunk 2]
```diff
@@ -2,14 +2,18 @@ import {createReadStream} from 'fs'
...
 ): Promise<Pick<StatsCompilation, 'assets' | 'chunks'>> {
   try {
     const path = resolve(process.cwd(), statsFilePath)
-    return parseChunked(createReadStream(path))  as StatsCompilation
-  } catch {
-    return {assets: [], chunks: undefined}
+    return (await parseChunked(createReadStream(path))) as StatsCompilation
+  } catch (error) {
+    if (error instanceof Error) {
+      core.warning(error)
```
Comment (mattcosta7) [inline src/parse-stats-file-to-json.ts:15]: annotation when the file isn't existing/can't be parsed

File: src/print-markdown.ts [hunk 3]
```diff
@@ -201,24 +184,13 @@ No files were changed`
...
     .map(chunkModule => printChunkModuleRow(chunkModule))
     .join('\n')}`
-  const detailedTable = `${DETAILED_CHUNK_TABLE_HEADERS}
-  ${modulesBySizeDescending
-    .map(chunkModule => printDetailedChunkModuleRow(chunkModule))
-    .join('\n')}`
 
   return `
 <details>
 <summary>**Changeset**</summary>
```
Comment (camchenry) [inline src/print-markdown.ts:190]: Minor thing: can we fix this header to remove the asterisks? It was supposed to be bold, but that doesn't work 😅

**Review by camchenry — APPROVED** [review]

File: src/print-markdown.ts (hunk 3 above)
Comment (mattcosta7) [reply src/print-markdown.ts:190]: next time - I had automerge on for this

**Pull request merged**